internal/configuration/configuration.go

```diff
@@ -121,3 +122,11 @@ func (m *Manager) GetConfigurationVersion() string {
 func (m *Manager) IsInitialConfig() bool {
 	return m.initialConfig.Load().(bool)
 }
+func (m *Manager) DeleteDeviceConfig() error {
+	log.Infof("Removing device config file: %s", m.deviceConfigFile)
+	err := os.Remove(m.deviceConfigFile)
+	if err != nil {
+		log.Error(err)
```

Comment:
should the error be ignored ? the expectation of  https://github.com/jakub-dzon/k4e-device-worker/pull/16/files#diff-8937ebacce65547ca6fcb202618610cee53b3011c0c6fa918ad383c5081004e4R50 is to get an error.

Reply:
you'r right fixed it